### PR TITLE
feat(configuration.base): hide sum feature

### PR DESCRIPTION
**Pull request by Willgeorges**
Possibilty for user to hide totals on cart view

### CODE REVIEW

File: lib/v1/configurations/items/settings.js [hunk 1]
```diff
@@ -8,6 +8,39 @@ module.exports = {
...
+              type: 'boolean',
+              default: false
+            }),
+            visibilty_of_sum: oneOf({
+              description: 'Specifies which sums are displayed on cart view if hide_sum_cart is enabled',
+              type: 'array',
+              items: {
+                type: 'string',
+                enum: [
+                  'Subtotal',
```
Comment (barbarafarias) [inline lib/v1/configurations/items/settings.js]: Maybe  "Subtotal" should also be non capitalised like the others?

Comment (barbarafarias) [reply lib/v1/configurations/items/settings.js]: Other then that LGTM

Comment (alghanor) [reply lib/v1/configurations/items/settings.js]: agree

**Pull request merged**